languages/python/oso/oso/roles.polar

```diff
@@ -0,0 +1,52 @@
+allow(actor, action, resource) if
+    resource(resource, namespace, actions, roles) and
+
+    # 'action' is valid for 'resource'
+    action in actions and
```

Comment:
make sure we have a test for a bad action

steve says validation will probs catch this

either way I think we can remove this check